testsuite/run_sets/testsuite.yml

```diff
@@ -23,17 +23,17 @@
   #         core_proxy_register_as_minion_with_script.feature
   #         core_proxy_register_as_minion_with_gui.feature
 - features/core_proxy_register_as_minion_with_gui.feature
-- features/core_proxy_branch_network.feature
+#- features/core_proxy_branch_network.feature
 # initialize clients
-- features/core_trad_register_client.feature
-- features/core_min_bootstrap.feature
-- features/core_min_salt_ssh.feature
-- features/core_centos_tradclient.feature
-- features/core_centos_salt.feature
-- features/min_centos_salt_install_package_and_patch.feature
-- features/core_centos_salt_ssh.feature
-#- features/core_ubuntu_salt.feature
```

Comment:
should not be commented

Reply:
Already discussed, we are in a development branch where we can just be focus in Ubuntu, we comment all the features no need, so the testsuite runs faster and only with important info.
That will be changed before we go to the upstream of ubuntu-support branch

Reply:
OK, looks Orion is adding some new features that requires CentOS, so I will just let all core features uncommented.